site/docs-md/guides/ionic-framework-app.md

```diff
@@ -0,0 +1,250 @@
+# Using Capacitor in an Ionic Framework App
+
+**Web Framework**: Ionic 4 + Angular  
+**Platforms**: Web, iOS, Android
+
+Capacitor makes it easy to build web apps that run natively on iOS, Android, desktop, and the web. In this guide, we'll add Camera functionality to an Ionic Angular app that works on the web, iOS, and Android. Ready to capture photos using just one JavaScript method call?  Let's get started.
+
+## Required Dependencies
+Building and deploying iOS and Android apps require additional dependencies, including an iOS and Android device. Please [follow the instructions here](../getting-started/dependencies) before continuing.
+
+## Prepare an Ionic App
+If you have an existing Ionic app, skip this section. If not, let's create an Ionic app first. In a Terminal, install Ionic (this also updates to the latest version of Ionic):
+
+```bash
+npm install -g ionic
+```
+
+Next, create a new Ionic app based on the "tabs" starter project:
+
+```bash
+ionic start capApp tabs
+```
+
+## Add Capacitor
```

Comment:
Ionic CLI has Capacitor integrations, so as you are starting with Ionic CLI, might be better to continue using it to add Capacitor, unless you want to focus on Capacitor CLI usage instead of Ionic CLI.
Example of starting the app with Capacitor already installed `ionic start capApp tabs --capacitor`
Example of adding Capacitor to a regular Ionic app you already created `ionic integrations enable capacitor`

Both of them do the npm install for core and cli and the init.

Reply:
excellent! didn't know that. will tweak